.github/workflows/screener-build.yml

```diff
@@ -97,31 +121,78 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile
 
+      - run: |
+          packageAffected=$(yarn --silent check:affected-package --packages @fluentui/docs --pr)
+          if [[ $packageAffected == false ]]; then
+            echo "Should skip screener"
+            echo "SKIP_SCREENER_BUILD=true" >> $GITHUB_ENV
+          else
+            echo "SKIP_SCREENER_BUILD=false" >> $GITHUB_ENV
+            echo "Should NOT skip screener"
+          fi
+        name: Check if northstar packages were affected (PR)
+        if: ${{env.ISPR == 'true'}}
+
+      - run: |
+          packageAffected=$(yarn --silent check:affected-package --packages @fluentui/docs)
+          if [[ $packageAffected == false ]]; then
+            echo "Should skip screener"
+            echo "SKIP_SCREENER_BUILD=true" >> $GITHUB_ENV
+          else
+            echo "SKIP_SCREENER_BUILD=false" >> $GITHUB_ENV
+            echo "Should NOT skip screener"
+          fi
+        name: Check if northstar packages were affected
+        if: ${{env.ISPR == 'false'}}
```

Comment:
Can we refactor this into a single block? Something like:

```
      - run: |
          echo "Is PR build? ${env.ISPR}"
          packageAffected=$(yarn --silent check:affected-package --packages @fluentui/vr-tests-react-components --pr=${env.ISPR})
          if [[ $packageAffected == false ]]; then
            echo "Should skip screener"
            echo "SKIP_SCREENER_BUILD=true" >> $GITHUB_ENV
          else
            echo "SKIP_SCREENER_BUILD=false" >> $GITHUB_ENV
            echo "Should NOT skip screener"
          fi
        name: Check if v9 packages were affected
```

(^ _note_ not sure that the actual code works, just to give you pointers)

Reply:
yes, very good idea, will do